MoEPluginBase/Manager/MoEPluginController.m

```diff
@@ -0,0 +1,248 @@
+//
+//  MoEPluginController.m
+//  MoEPluginBase
+//
+//  Created by Rakshitha on 09/12/21.
+//
+
+#import <Foundation/Foundation.h>
+#import "MoEPluginController.h"
+#import "MoEPluginMessage.h"
+#import "MoEPluginUtils.h"
+#import "MoEngage/MoEngage.h"
+#import "MOInAppCampaign+Utility.h"
+#import "MoEPluginConstants.h"
+#import "MoEPluginMessageQueueHandler.h"
+#import "MoEngage/MoEngage.h"
+#import "MoEPluginCoordinator.h"
+@import MoEngageObjCUtils;
+
+@interface MoEPluginController()<MOInAppNativDelegate, MOMessagingDelegate>
+@property(nonatomic, strong) MoEPluginMessageQueueHandler* messageHandler;
+
+
+@end
+
+@implementation MoEPluginController
+
+- (instancetype)init {
+    self.messageHandler = [[MoEPluginMessageQueueHandler alloc]init];
+    return self;
+}
+
+-(void) flushMessageQueue {
+    [self.messageHandler flushMessageQueue];
+}
+
+-(void)queueMessage:(MoEPluginMessage*)message {
+    [self.messageHandler queueMessage:message];
+}
+
+- (void)initializeSDKWithConfig:(MOSDKConfig*)sdkConfig isSecondaryInstance:(BOOL)isSecondary andLaunchOptions:(NSDictionary*)launchOptions {
```

Comment:
Let's have a flag name as `isDefaultInstance` and change the checks accordingly. isSecondary didn't feel right, hence we changed the initializer method names native SDK as well.